ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/connected_endpoints_keys_key.yml

```diff
@@ -0,0 +1,18 @@
+type: l3leaf
+l3leaf:
+  defaults:
+    loopback_ipv4_pool: 192.168.255.0/24
+    vtep_loopback_ipv4_pool: 192.168.254.0/24
+  nodes:
+    - name: connected_endpoints_keys_key
+      id: 101
+      bgp_as: 101
+
+servers:
```

Comment:
```suggestion
# Testing "servers" as dict of dict
# Dict type is deprecated in 4.0.0. To be removed in 5.0.0
servers:
```

Reply:
Done!

Reply:
Descriptions like this should be put in above _every_ dict-of-dict with the wildcard keys, so we can quickly identify what is being tested.

Reply:
Done